modules/nf-core/last/dotplot/main.nf

```diff
@@ -8,8 +8,9 @@ process LAST_DOTPLOT {
         'biocontainers/last:1542--h43eeafb_1' }"
 
     input:
-    tuple val(meta), path(maf)
+    tuple val(meta), path(maf), path(annot_b)
     val(format)
+    path(annot_a)
```

Comment:
I feel this probably wants a meta map, and I'd put it above the val(format) channel.

Reply:
Done in ad87c1c3cf0ae7ccd01b7b18cf1c6f3d06e92b9d